pynucastro/networks/rate_collection.py

```diff
@@ -127,6 +127,11 @@ def set_solar_like(self, Z=0.02):
 
         self.normalize()
 
+    def set_array(self, arr):
+        """ set all species from an iterable (in no particular order)"""
```

Comment:
`arr` is accessed as a Sequence rather than an Iterable. Specifying a consistent order is nice, as I think we can use this elsewhere too.
```suggestion
        """ set all species from a sequence of mass fractions, in the same
        order as returned by get_nuclei() """
```